lua/wire/client/text_editor/texteditor.lua

```diff
@@ -1154,14 +1159,16 @@ function EDITOR:CountFinds( str )
 	if whole_word_only then
 		local pattern = "([^a-zA-Z0-9_])"..str.."([^a-zA-Z0-9_])"
 		txt = " " .. txt
-		local num1, num2 = 0, 0
+		local num1, num2
 		txt, num1 = txt:gsub( pattern, "%1%2" )
 		if txt == "" then return num1 end
-		txt, num2 = txt:gsub( pattern, "%1%2" )
+		local _
+		_, num2 = txt:gsub( pattern, "%1%2" )
```

Comment:
`num2 = select(2, txt:gsub(pattern, "%1%2"))` might look a bit better here

Reply:
or `num2 = {whatever}[2]` idk which is faster

Reply:
That has the overhead of table creation. Also as a bonus `select` is compiled when first argument is a constant.